props/blaster_BC_buttons.h

```diff
@@ -649,7 +649,7 @@ void Loop() override {
   }
 
   // Blaster effects, auto fire is handled by begin/end lockup
-  void SB_Effect(EffectType effect, float location) override {
+  void SB_Effect2(EffectType effect, float location) {
```

Comment:
Still needs "override"

Reply:
oh, I see now what the issue was. It wasn't compiling, said it was marked override but didn't override. Seemed odd since it's overriding already in the Blaster class and we're inheriting from there. But I found it worked as SB_Effect2, so thought that was the right move.
The actual issue was the addition of EffectLocation since this was last worked on. changing float to EffectLocation makes things right now.
If I understand correctly, it only "needs" to be SB_Effect2 if we were trying to get sound lengths for example.

Reply:
Correct.
Generally SB_Effect() is the right place to play sounds, and SB_Effect2() is the right place to do something after that.